style.css

```diff
@@ -0,0 +1,768 @@
+  color: tomato;
+  padding-top: 16%;
+  padding-left: 8%;
+  padding-bottom: 2%;
+}
+
+.intro-title-two {
+  -webkit-background-clip: text;
+  background-image: url("images/orangebg.jpg");
+  color: transparent;
+  font-weight: 700;
+  font-size: 30px;
+  padding-top: 3%;
+  padding-left: 8%;
+}
+
+.explanation-paragraph {
+  border: 1px solid #fff;
+  background-color: #fff;
+  border-radius: 1rem;
+  box-shadow: 3px 2px 2px #e4e4e4;
+  width: 90%;
+  padding: 5%;
+  font-family: 'Lato', sans-serif;
+}
+
+.event-date {
+  font-family: 'Lato', sans-serif;
+  font-size: 24px;
+  font-weight: 700;
+  font-style: normal;
+  color: #292929;
+}
+
+.copyright {
+  font-family: 'Lato', sans-serif;
+  font-size: 18px;
+  font-weight: 300;
+  font-style: normal;
+  color: #292929;
+}
+
+#packages-offered {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 20px 10px;
+  width: auto;
+  height: auto;
+  gap: 15px;
+  background-image: url('images/icebackgr.png');
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.packages-title {
+  padding: 0 15%;
+  margin-bottom: 4%;
+  font-family: 'Lato', sans-serif;
+  font-size: 14px;
+  font-weight: 700;
+  text-align: center;
+  text-decoration-line: underline;
+  text-decoration-color: #ec5242;
+  text-underline-offset: 10px;
+  color: #fff;
+}
+
+.card-stack {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: auto;
+  height: auto;
+  gap: 10px;
+  padding: 0;
+}
+
+.card-stack .cards {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: 12px;
+  padding: 5px 10px;
+  background-image: url("images/icebackgr.png");
+  font-family: 'Lato', sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  font-style: normal;
+}
+
+.cards-img {
+  box-sizing: border-box;
+  display: flex;
+  width: 44px;
+  height: 44px;
+}
+
+.card-title {
+  display: flex;
+  font-family: 'Lato', sans-serif;
+  font-weight: 700;
+  font-size: 20px;
+  text-align: left;
+  color: #ec5242;
+  width: 50px;
+}
+
+.card-text {
+  font-family: 'Lato', sans-serif;
+  text-align: left;
+  width: 150px;
+  color: #fff;
+  padding-left: 40px;
+}
+
+.view-all {
+  display: none;
+}
+
+.join-us {
+  width: 70%;
+  height: 10%;
+  font-size: 14px;
+  font-weight: 400;
+  color: #fff;
+  margin-top: 5%;
+  padding: 3% 5%;
+  background-color: #ec5242;
+  border-radius: 0.5rem;
+}
+
+.speakers-maintitle {
+  font-family: 'Lato', sans-serif;
+  font-size: 16px;
+  font-weight: 900;
+  font-style: normal;
+  text-align: center;
+  padding: 20px 10px 0 10px;
+  color: #292929;
+  text-decoration-line: underline;
+  text-decoration-color: #ec5242;
+  text-underline-offset: 10px;
+  margin-right: 15%;
+}
+
+#speaker-cards {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  height: auto;
+  width: auto;
+  margin: 50px;
+}
+
+.each-card {
+  display: flex;
+  gap: 2%;
+  padding: 0;
+}
```

Comment:
- [ ] Hey Marianna, you're doing a really great job so far and you're almost done but as mentioned previously, the content inside the speakers' card is not aligned correctly as shown in the original design. Please work on that as one of the objectives of this module is to ensure you can implement a given design properly.
- Your design:
  ![image](https://github.com/MariamOsman11/CAPSTONE-PROJECT-ONE/assets/104968236/6edf938f-18fc-4710-8d3d-3da6c6d6df41)
- Behance design:
  ![image](https://github.com/MariamOsman11/CAPSTONE-PROJECT-ONE/assets/104968236/07ef7ca2-b892-4398-b382-0b76d42f17a0)